common/tee.edl

```diff
@@ -14,14 +14,19 @@
 
 enclave
 {
+    /* The end goal is to publish all EDL as part of the SDK. At that time
+     * enclave developers will simply import the edl files/functions they
+     * need and this file will no longer be required. These files are imported
+     * as a convenience to allow refactoring of system EDL files without the need
+     * to add a new build definition for each file.
+     */
+    from "logging.edl" import *;
+    from "memory.edl" import *;
+
     include "openenclave/bits/types.h"
 
     trusted
     {
-        public void oe_log_init_ecall(
-            [in, string] const char* enclave_path,
-            uint32_t log_level);
-
         public oe_result_t oe_verify_report_ecall(
```

Comment:
This should probably be spun out into an attestation.edl or similar at some point ...

Reply:
I almost did that but then I wasn't sure which ones were related to attestation and which weren't. Would you say oe_verify_report_ecall is attestation-related but the get_public_key functions are not?

I can go ahead and do that in this PR

Reply:
> Would you say oe_verify_report_ecall is attestation-related but the get_public_key functions are not?

Yes, exactly.

Reply:
I had equated oeenclave functionality with attestation, but you're right  that we should treat key handling functions as not being related to attestation. The desire of the comment was to have tee.edl be a wrapper that can just go away when all the nested functions are done, so we could split the resulting set into attestation.edl (verify report, which I suspect should get subsumed into the attestation plugin model) and tee_keys.edl.

Reply:
I split into `attestation.edl` and keys.edl`